src/runtime/node/crypto/$cloudflare.ts

```diff
@@ -106,18 +105,21 @@ export const {
   timingSafeEqual,
 } = workerdCrypto;
 
+// Special case getRandomValues as it must be bound to the webcrypto object
+export const getRandomValues = workerdCrypto.getRandomValues.bind(
+  workerdCrypto.webcrypto,
+);
```

Comment:
This is the correct fix.
We need to ensure that every export of this function is correctly bound.
Not just the one from webcrypto.